docs/src/main/asciidoc/extension-writing-dev-service.adoc

```diff
@@ -5,118 +5,149 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 [id="extension-writing-dev-service"]
 = Writing a Dev Service
+
 include::_attributes.adoc[]
 :categories: writing-extensions
 :diataxis-type: howto
 :topics: extensions
-////
-////
 
+Learn how to develop a xref:dev-services.adoc[Dev Service] for your extension in order to replace an external service in development mode.
 
 == Prerequisites
 
-- You should already have an xref:building-my-first-extension.adoc[extension structure] in place
-- You should have a containerised version of your external service (not all Dev Services rely on containers, but most do)
+:prerequisites-time: 15 minutes
+:prerequisites-docker:
+:prerequisites-no-cli:
+:prerequisites-no-graalvm:
+include::{includes}/prerequisites.adoc[]
+* An xref:building-my-first-extension.adoc[extension structure] in place
+* A containerised version of your external service (not all Dev Services rely on containers, but most do)
 
 == Creating a Dev Service
 
-If your extension provides APIs for connecting to an external service, it's a good idea to provide a xref:dev-services.adoc[Dev Service] implementation.
+If your extension provides APIs for connecting to an external service, it's a good idea to provide a dev service implementation.
+
+First, you must add the following dependency to your build file :
```

Comment:
I think we need to be more precise here. It has to be added to the build part of the `deployment` module.

Reply:
Perfect, thanks.